mixer/pkg/attribute/mutableBag.go

```diff
@@ -117,6 +117,9 @@ func copyValue(v interface{}) interface{} {
 
 // Done indicates the bag can be reclaimed.
 func (mb *MutableBag) Done() {
+	if mb == nil {
```

Comment:
Ah... This is more tricky than needed, however. If you call Done on a bag that is not nilled, it will panic due to the error below. If you call Done on a bag that has not been nilled, it will work. I think we should stick to one way or another.

Reply:
removed.